types/youtube-dl/tsconfig.json

```diff
@@ -0,0 +1,25 @@
+{
+    "files": [
+        "index.d.ts",
+        "youtube-dl-tests.tsx"
+    ],
+    "compilerOptions": {
+        "module": "commonjs",
+        "lib": [
+            "es6",
+            "dom"
+        ],
+        "noImplicitAny": true,
+        "noImplicitThis": true,
+        "strictNullChecks": true,
+        "strictFunctionTypes": false,
```

Comment:
This should be `true` for all new packages.

Reply:
done